libethereum/State.cpp

```diff
@@ -518,6 +518,11 @@ std::pair<ExecutionResult, TransactionReceipt> State::execute(EnvInfo const& _en
 		onOp = Executive::simpleTrace(); // override tracer
 #endif
 
+	BlockHeader bh;
+	bh.setNumber(_envInfo.number());
+	bh.setGasLimit(_envInfo.gasLimit());
+	_sealEngine.verifyTransaction(ImportRequirements::Everything, _t, bh);
```

Comment:
Most of the checks inside `verifyTransaction` are duplicated in `Executive::initialize` I think

So I think we should better move this `verifyTransaction` call into `Executive::initialize`, removing the checks there.
I'm not sure if it's easy, when I tried to do this refactoring, I think saving the exception to `m_excepted` was something hard to deal with, then I gave up.

But just adding the call to `verifyTransaction` here doesn't seem right